client/src/main/java/crawlercommons/urlfrontier/client/GetURLStatus.java

```diff
@@ -0,0 +1,101 @@
+package crawlercommons.urlfrontier.client;
+
+import crawlercommons.urlfrontier.URLFrontierGrpc;
+import crawlercommons.urlfrontier.URLFrontierGrpc.URLFrontierBlockingStub;
+import crawlercommons.urlfrontier.Urlfrontier.URLItem;
+import crawlercommons.urlfrontier.Urlfrontier.URLStatusRequest;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import picocli.CommandLine.ArgGroup;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.ParentCommand;
+
+@Command(name = "GetURLStatus", description = "Get the status of an URL")
+public class GetURLStatus implements Runnable {
+
+    @ParentCommand private Client parent;
+
+    @ArgGroup(exclusive = false, multiplicity = "1")
+    Dependent group;
+
+    static class Dependent {
+        @Option(
+                names = {"-c", "--crawlID"},
+                required = true,
+                paramLabel = "STRING",
+                description = "crawl of the url to be checked")
+        private String crawl;
+
+        @Option(
+                names = {"-k", "--key"},
+                required = true,
+                paramLabel = "STRING",
+                description = "key to use to target a specific queue")
+        private String key;
+
+        @Option(
+                names = {"-u", "--url"},
+                required = true,
+                paramLabel = "STRING",
+                description = "url to check for")
+        private String url;
+    }
+
+    @Option(
+            names = {"-p", "--parsedate"},
+            description = "Print the Epoch refetch time in local time")
```

Comment:
maybe explain that otherwise the value displayed is seconds of UTC time since Unix epoch

Reply:
OK